src/configure.js

```diff
@@ -99,9 +103,20 @@ const prompt = async function() {
             type: 'confirm',
             message: 'Would you like WP Local Docker to manage your hosts file?',
             default: currentHosts !== undefined ? currentHosts : defaults.manageHosts,
-        },
+        }
     ];
 
+    if ( fs.exists( path.join( getConfigDirectory(), 'global' ) ) ) {
+        questions.push(
+            {
+                name: 'overwriteGlobal',
+                type: 'confirm',
+                message: 'Do you want to reset your global services configuration? This will reset any customizations you have made.',
+                default: false
+            }
+        );
+    }
```

Comment:
We can use `when` to increase readability:
```
	const questions = [
		...
        {
            name: 'overwriteGlobal',
            type: 'confirm',
            message: 'Do you want to reset your global services configuration? This will reset any customizations you have made.',
			when: await fs.exists( path.join( getConfigDirectory(), 'global' ) ),
            default: false
        }
    ];

    const answers = await inquirer.prompt( questions );

```